guides/common/modules/proc_disabling-registration-to-insights.adoc

```diff
@@ -1,8 +1,8 @@
 [id='disabling-registration-with-insights_{context}']
 
```

Comment:
```suggestion
```

Reply:
Fixed. Also updating title to "Disabling Red Hat Insights registration" because "registration to RH Insights" looks weird. Normally, one registers **for** 'x', not **to**.